### PR TITLE
Jetpack Connect: add Jetpack branding to plans grid

### LINKED ISSUE #28419: Update page branding during Jetpack Connection flow
The Jetpack plan selection step of the connection flow exists on a WordPress.com branded view. Screenshot: https://cloudup.com/i9lIe5Vymx9

Let's update this to reflect Jetpack branding so it's clearer for our self-hosted users that this is part of the Jetpack experience.

screen shot 2018-07-12 at 3 34 23 pm

### PR DESCRIPTION
**Pull request by keoshi**
#### Changes proposed in this Pull Request

* Adds Jetpack logo to plans grid.
* Adds Jetpack brand colors to buttons.

#### Testing instructions

* Start on `/jetpack/connect/store/`
* Make sure you see the Jetpack logo and the Jetpack brand colors on the plans grid.

#### Before

image

#### After

image

Fixes #28419

### CODE REVIEW

**Comment by simison** [comment]
Noting that two active primary buttons (instead of just one) are a known issue: https://github.com/Automattic/wp-calypso/issues/28882

**Comment by keoshi** [comment]
Yes, it came up yesterday on Slack too. Thanks so much for noting that @simison!

**Comment by rickybanister** [comment]
Looks good—the screenshots show the plans page grid vs the connection flow grid. Is this change reflected there as well? If so does adding the logo make the page height issue worse or is that not an issue if we also merge a solution to the hidden 'start with free' button?

File: client/jetpack-connect/plans-grid.jsx [hunk 1]
```diff
@@ -44,9 +45,10 @@ class JetpackPlansGrid extends Component {
 
 	render() {
 		return (
-			<Main wideLayout className="jetpack-connect__hide-plan-icons">
+			<MainWrapper isWide className="jetpack-connect__hide-plan-icons">
```
[the remark this answers is not in the document]
Comment (keoshi) [reply client/jetpack-connect/plans-grid.jsx]: Hmm, good catch — thanks for that, Jon. Are we still good to go with this PR as it stands?

[the remark this answers is not in the document]
Comment (sirreal) [reply client/jetpack-connect/plans-grid.jsx]: Absolutely, I've approved. This is a note I'd like to get addressed in the future.

**Comment by adamziel** [comment]
#28307 is now merged

**Pull request merged**